SalesforceCommerceCloud/pwa-kit
# Always enable the fav icon in product tiles

# Description



There was a bug with wishlist feature where the favourite icons on product tiles are not visible to guest users. This PR fixes the issue by enabling the feature on those product tiles.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- pass `enableFavourite: true` to product tile component

# How to Test-Drive This PR

- `npm start` and go to http://localhost:3000/en-GB/category/womens-clothing-dresses, should see the heart empty icons shown on product tiles.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
